Pacifique78/MyDiary
# #169171219 update es-lint to use Airbnb

#### What does this PR do?
This PR updates es-lint configurations to use Airbnb and fix the tests.


#### Description of Task to be completed
>- initialize es-lint configuration file for new.
>- update the expired tokens
>- correct errors caused by updating es-lint

#### How should this be manually tested?
>- Clone this repo 
>- run `npm run lint` to check linting errors.
>- run `npm test` to check if the tests are passing.
>- check if Travis tests are passing.

#### What are the relevant pivotal tracker stories?
#169171219

#### Screenshots (if appropriate)
![Screenshot (74)](https://user-images.githubusercontent.com/50268181/67607982-a9ebec00-f786-11e9-86f4-1dfa7ae343ec.png)
